iot-hub/Samples/device/TwinSample/TwinSample.csproj

```diff
@@ -3,10 +3,11 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.*" />
+	<PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.28.0" />

```

Comment:
Why aren't we leaving this as "1.*" to always get the latest?

Reply:
Because in .NET that is not considered a best practice. The samples could possibly break without us knowing. We should be explicit about our package updates.

Reply:
Also, we don't want to imply that a customer should do that in their project, either. Again, being a bad practice in .NET.